.pre-commit-config.yaml

```diff
@@ -98,7 +98,7 @@ repos:
         entry: flake8-rst
         types: [rst]
         args: [--filename=*.rst]
-        additional_dependencies: [flake8-rst==0.7.0, flake8==3.7.9]
+        additional_dependencies: [flake8-rst==0.7.0, flake8==4.0.1]
```

Comment:
you don't need to upgrade this one

Reply:
this is just a dependency for `flake8-rst`, which seems to be broken for modern versions of flake8

we should probably replace it, but for now it's OK to keep it with flake8 pinned

Reply:
Ok, sounds good. I reverted it.